### PR TITLE
Pill-Box Implementation

### PR DESCRIPTION
**Pull request by Polsander**
Opened this PR for Juan so that other issues can be unblocked and can start being worked on.

This PR oversees the implementation of bread crumbs and pill buttons at the top of the table pages.

### CODE REVIEW

File: microservices/client/src/components/table/CCGTable.tsx [hunk 1]
```diff
@@ -69,6 +71,24 @@ export const CCGTable: React.FC<ICCGTableProps> = (props) => {
 
   return (
     <div>
+      <h5>
+        <Breadcrumb>
+          <Breadcrumb.Item href='#"'><MdOutlineHome /></Breadcrumb.Item>
```
Comment (Polsander) [inline microservices/client/src/components/table/CCGTable.tsx:76]: You can link this to home, does not have to be to `#`

File: microservices/client/src/components/buttons/PillButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { IconType } from 'react-icons';
+
+
+
+interface PillButtonProps {
```
Comment (Polsander) [inline microservices/client/src/components/buttons/PillButtons.tsx:6]: convertion for definiting interfaces is always to start with an `I` in front
```suggestion
interface IPillButtonProps {
```

**Comment by Polsander** [comment]
image
^This is your implementation
image
^This is our Figma

Just being a bit nit picky.
Overall size of the pill boxes are fine if we are following spec, but the font looks like it's bolded for some reason especially in bread crumbs and title. Global styles should take care of our font style by default, so double check your font-weight.

`Customers` also looks way too big.

I am also more of a fan of your bread-crumbs than what we have in the figma implementation, so we can keep that.
Icons can also be difficult to work with, but see if you can align them better with the text if possible (if not, no big deal).

**Comment by Polsander** [comment]
@juanitosol Please address all concerns and ping me and Fernando to review.

File: microservices/client/src/components/table/CCGTable.tsx [hunk 3]
```diff
@@ -69,6 +71,24 @@ export const CCGTable: React.FC<ICCGTableProps> = (props) => {
...
+          <Breadcrumb.Item active>Customers</Breadcrumb.Item>
+        </Breadcrumb>
+      </h5>
+      <h2>
+        <div className='container1'>
+          <span className='font-48'>Customers</span>
+          <div className='button-container'>
+            <PillButtons1 label="Export" icon={MdOutlinePictureAsPdf} onClick={() => null} />
```
[the remark this answers is not in the document]
Comment (juanitosol) [reply microservices/client/src/components/table/CCGTable.tsx:84]: Removed

**Comment by Polsander** [comment]
Not a fan of the implementation. Since Juan is now off the project, I will take over this as a personalized ticket. Closing the PR.

**Pull request closed without merging**